Python/Product/Profiling/PythonProfilingPackage.cs

```diff
@@ -303,9 +316,75 @@ protected override int CreateToolWindow(ref Guid toolWindowType, int id)
             config.ScriptArguments = runTarget.Arguments;
             config.WorkingDirectory = runTarget.WorkingDirectory;
 
-            RunProfiler(session, config, openReport);
+#if EXTERNAL_PROFILER_DRIVER
+            if (useVTune) {
+                RunVTune(session, config, openReport);
+            } else {
+#endif
+                RunProfiler(session, config, openReport);
+#if EXTERNAL_PROFILER_DRIVER
+            }
+#endif
         }
 
+#if EXTERNAL_PROFILER_DRIVER
+        private static void RunVTune(SessionNode session, LaunchConfiguration config, bool openReport) {
+#if false
+            var pyexe = ProcessOutput.QuoteSingleArgument(stndTarget.InterpreterPath);
+
+            if (pyexe == string.Empty) {
+                if (stndTarget.PythonInterpreter != null) {
+                    var registry = session._serviceProvider.GetComponentModel().GetService<IInterpreterRegistryService>();
+                    var interpreter = registry.FindConfiguration(runTarget.PythonInterpreter.Id);
+                    if (interpreter == null) { 
+                        /* ??? */;
+                        MessageBox.Show("Could not find interpreter in the registry");
+                    } else {
+                        MessageBox.Show($"The Python interpreter in question is: [{interpreter}]");
+                    }
+                }
+            }
+#endif
+
+            var interpreter = config.GetInterpreterPath();
+            if (!File.Exists(interpreter)) {
+                MessageBox.Show("Can't find specified python interpreter.");
+                return;
+            }
+
+            string outPathDir = Path.GetTempPath();
```

Comment:
Every running instance of VS will stomp on each other, because GetTempPath() always returns the same folder. You'll need to create an instance specific subfolder under outPathDir to avoid collision.

ex: everyone is writing to `c:\users\huvalo\apdata\local\temp\sample.dwjson`

Reply:
Would you suggest of a way to get the VS instance id?  Or a reasonable proxy?  Thanks!